utils/vscode/src/lsp/jsonrpc2/handler.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
```

Comment:
I believe this would be the first time a BSD-licensed file would be in the SPIRV-Tools repository itself.
The rest has been Apache 2.0

Reply:
Changed to Apache 2.0

Reply:
This and other files still say "BSD-style license", and then reference the LICENSE file.
I'd like to see the standard Apache2 header everywhere instead.